src/applications/travel-pay/components/submit-flow/pages/AddressPage.jsx

```diff
@@ -1,28 +1,174 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 
-import { VaButtonPair } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import {
+  VaButtonPair,
+  VaRadio,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import { focusElement, scrollToTop } from 'platform/utilities/ui';
+
+import { HelpTextGeneral, HelpTextModalities } from '../../HelpText';
+
+const AddressPage = ({
+  address,
+  pageIndex,
+  setPageIndex,
+  yesNo,
+  setYesNo,
+  setIsUnsupportedClaimType,
+}) => {
+  useEffect(
+    () => {
+      scrollToTop('topScrollElement');
+      if (!address) {
+        focusElement('h1');
```

Comment:
Is the aim behind this to set the scroll height at a specific point or to make the screen reader call out the heading each time the view changes? I'm wondering if `aria-live` might help address the latter aim.
I was under the impression that focusing non-interactive elements was something to be avoided.